packages/pools/src/router/routes.ts

```diff
@@ -31,10 +31,6 @@ export type OptimizedRoutesParams = {
   /** **Ordered** IDs of pools to be prioritized in route selection.
    *  The first pools in the array will be selected first. */
   preferredPoolIds?: string[];
-  /** IDs of pools receiving OSMO incentives. */
-  incentivizedPoolIds: string[];
-  /** Min/base denom of stake currency of chain (OSMO) */
-  stakeCurrencyMinDenom: string;
```

Comment:
Is this a drive-by change?

Reply:
I forgot to remove it when I removed the OSMO 2 hop discount, as that's all it was used for. So yes you could say it's drive by.